gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/SettingsGradingSchemaPanel.java

```diff
@@ -300,3 +304,23 @@ public int compare(final String o1, final String o2) {
 		}
 	}
 }
+/**
+ * Comparator to ensure correct ordering of percents
+ */
+class PercentComparator implements Comparator<String>, Serializable {
```

Comment:
I've split all of these comparators out into their own classes. Could you do the same?